databases/week3/meal_sharing.sql

```diff
@@ -0,0 +1,290 @@
+CREATE DATABASE meals;
+SET
+  NAMES utf8mb4;
+USE meals;
+CREATE TABLE `meal`
```

Comment:
Maybe put the table creations in a separate file to make it easier to get an overview of db structure vs the queries you are making